src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -446,6 +446,11 @@ public void download() {
                     linkedFiles.set(oldFileIndex, newLinkedFile);
                 }
                 entry.setFiles(linkedFiles);
+                // Notify in bar when the file type is HTML.
+                if (newLinkedFile.getFileType().equals(StandardExternalFileType.URL.getName())) {
+                    dialogService.notify(Localization.lang("Downloaded website as an HTML file."));
+                    LOGGER.debug("Downloaded website {} as an HTML file at {}", linkedFile.getLink(), destination);
```

Comment:
The thing is you can also link a html file and really want to download the html. Thus it is not always a mistake to download html. Maybe it's ok to ignore this problematic (except if the user sets the file type to HTML)?

Apart from this, I also like the suggestion of @Krzmbrzl in https://github.com/JabRef/jabref/issues/7452
> My suggestion for a mitigation would be to check the downloaded file and if it starts with plain text <!DOCTYPE html>, then assume the download has failed and remove the "PDF" again (restoring the file link with the download URL in JabRef).

to keep the link in case the download produced a html file. In this case, the user can activate his university vpn/proxy and simply try again.